attribute-event-requests/FAM-event.md

```diff
@@ -85,3 +86,33 @@ In [GEDCOM X](https://github.com/FamilySearch/gedcomx/blob/master/specifications
 ## Marriage
 
 In [GEDCOM X](https://github.com/FamilySearch/gedcomx/blob/master/specifications/fact-types-specification.md) with URI `http://gedcomx.org/Marriage`
+
+## Separation
+
+### Description
+
+*proposed description missing*
+
+*In [GEDCOM X](https://github.com/FamilySearch/gedcomx/blob/master/specifications/fact-types-specification.md) as* "A fact of a couple's separation."
+
+### Value
+
+Found in the following historical records:
+
+- (records not yet identified)
+
+### Absence
+
+There is no other structure that covers this event, as it can apply to couples that cohabitate and are not married as well as to those who are married in some manner. And for those who are married, some couples may separate and never actually file for or go through with a formal divorce.
+
+### Used
+
+- Part of the [GEDCOM X specification](https://github.com/FamilySearch/gedcomx/blob/master/specifications/fact-types-specification.md) with URI `http://gedcomx.org/Separation`
+
+- Used by the [FamilySearch API](https://www.familysearch.org/developers/docs/guides/facts) via GEDCOM X.
+
```

Comment:
```suggestion
```

This event type is not documented in the FamilySearch API (at least not at the link provided). To the best of my knowledge, it is also not used by any FamilySearch application (I'm not a FamilySearch employee, but so far as I know they only use their API, not GEDCOM-X, in their products)

Reply:
Oops, yes you are correct I should have done due diligence. Fixed.